app/app-www/js/global.js

```diff
@@ -0,0 +1,18 @@
+define([
```

Comment:
So... we kind of talked about injecting astro classes to the html tag already & had implemented a solution using `astro-client` but since then, we've undone that.

This is just pulling in a temporary solution that we did for extra so that bundled app content would be app aware... but really, this is a global script that we can let run on all bundled app content.

Currently it's marked as a dependency in the `app-www/js/config.js` folder which will make sure this script runs so long as the HTML file includes require js.

Downside to this is that classes are injected AFTER css has been pulled in, but for now this is a temporary solution.